src/octopeer-github/test/content/ElementEventBinding/ScrollElementEventBindingTest.ts

```diff
@@ -0,0 +1,59 @@
+import Spy = jasmine.Spy;
+/**
+ * Created by Mitchell on 25-5-2016.
+ * Behaviour unit tests for the ScrollIntoViewElementEventBinding class.
+ */
+
+const esbFactory = new ElementSelectionBehaviourFactory();
+const eebFactory = new ElementEventBindingFactory();
+
+let binder: ElementEventBinding;
+let database: DatabaseAdaptable = new ConsoleLogDatabaseAdapter();
+let selector: ElementSelectionBehaviour;
```

Comment:
They are instantiated inside the tests, so that can still cause problems.
I agree that the factories will not cause much trouble cause (at the moment) there is not much that can go wrong when creating new tests. But you can better be safe than sorry when someone changes the architecture.